### ISSUE TITLE
Plot2Dynmap not showing who owns the plot

### ISSUE THREAD

**Issue by TNTUP** [IntellectualSites/PlotSquared#2402]
__*NOTICE: Bukkit/Spigot versions 1.7.10 to 1.12.2 are considered legacy and will receive limited support. Please consider upgrading to 1.13 for future support. Plugins exist for 1.13+ that bring back old behaviors found in 1.8*__
# Bug report template 

**[REQUIRED] PlotSquared Version Number:** 
plot version
[0 20:07:37 INFO]: [P2] >> PlotSquared (Version: PlotSquared-4.292)
[0 20:07:37 INFO]: [P2] >> Authors: Citymonstret & Empire92 & MattBDev & dordsor21
[0 20:07:37 INFO]: [P2] >> Wiki: https://github.com/IntellectualCrafters/PlotSquared/wiki
[0 20:07:37 INFO]: [P2] >> Newest Version: development-builds

**[REQUIRED] Spigot/Paper Version Number:** 
ver
[0 20:08:04 INFO]: This server is running Paper version git-Paper-"17fe7a1" (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
[0 20:08:04 INFO]: Previous version: git-Paper-"fb25dc1" (MC: 1.13.2)
[0 20:08:04 INFO]: Checking version, please wait...
[0 20:08:04 INFO]: You are 8 version(s) behind

**[REQUIRED] Minecraft Version Number:** 
1.13.2

**Links to worlds.yml file and settings.yml file:** 
worlds.yml --> https://pastebin.com/HWsWg96u
settings.yml --> https://pastebin.com/JqFQr9WK

**[REQUIRED] Description of the problem:** 
Using Plot2Dynmap doesn't show who owns the plot and throws an error. No updates from the dev on spigotmc or from this issue #2193 

**Any relevant console output or screenshots:**
https://pastebin.com/nYs21900

**Plugins being used on the server:**
Can be given in private

**How to replicate:** 
Download Plot2Squared-Breaking and Plot2Dynmap provided from Sauilitired https://github.com/IntellectualSites/PlotSquared/issues/2193#issuecomment-450006547

**Checklist**:

- [x] I included all information required in the sections above
- [] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of PlotSquared
- [] I made sure the bug/error is not caused by any other plugin

Comments: 
There's already an issue about Plot2Dynmap but I guess that one got abandoned so incase if this issue counts as a duplicate, feel free to close it. (didn't check the second one)

I could've reported the issue in the Plot2Dynmap issue tracker, It got a duplicate and got closed.  (Let me know I'll close this issue and report it to the Plot2Dynmap issue instead)

**Comment by AeSix** [comment]
Is this the case when using just P2,  P2dm and Dynmap?
It would be helpful to the devs if you could test using just those three plugins. 

Additionally, check for a different version of Dynmap, as it may also be the cause.

**Comment by TNTUP** [comment]
Even, both yes. I'm on the "latest" build of dynmap, according to the spigotmc website, Dynmap v3.0-beta-4 currently.

**Comment by AeSix** [comment]
Have you tried a version before 1.14 support was added? 
This would be 3.0 beta 3, from spigotmc, according to the updates. 
It's completely possible he broke something for 1.13 when fixing it for 1.14.

**Comment by TNTUP** [comment]
Last time I checked, I was on beta 3 of Dynmap, PlotSquared-breaking-277 and Plot2Dynmap (link provided above). Before 1.14, I don't think so, I've started migrating Creative world into a server and updating it to 1.13.2 (1.14.2 was in a snapshot state, I think it was at 1.14.1 release)

**Comment by AeSix** [comment]
Is this still an issue with all updated software in question?
If it is no longer an issue, please close the ticket.   If it is, please update with appropriate information :)

**Comment by TNTUP** [comment]
Yes, no luck and no updates either from the developer

**Comment by AeSix** [comment]
Dynmap has been updated.  If it's still not working with 1.13.2/1.14 PlotSquared, you'll have to open an issue with mikeprimm.  https://www.spigotmc.org/resources/dynmap.274/  He may need to work with the P2 devs to sort the issue.

**Comment by TNTUP** [comment]
Closing ticket without proper testing? Just tried Dynmap 3.0 beta5 and still the same error as before. Don't think mikeprimm would ever fix/check this problem because he don't manage Plot2Dynmap. So guessing this plugin is now dead for 1.13.x servers

**Comment by AeSix** [comment]
So, I learned this morning that P2D has it's own github repo, and that only one of the active devs has access to it.  Therefore, this is the wrong place to file issues for P2D, and should instead be filed at:
https://github.com/boy0001/plot2dynmap/issues

Dynmap is a plugin which must correct itself to work with other plugins.  Not the other way around.  However, P2D does this, as a bridging plugin for both P2 and Dynamp.  

tl;dr:  Plot2Dynamp is not a main component of PlotSquared. This is the wrong issue tracker.  See above.

**Comment by TNTUP** [comment]
It is still somewhat valid because, the developer who forked Plot2Dynmap is IntellectualSites which is the current one working with 1.13.x and 1.14.x (not tested yet). That's why I created a ticket here to see if there's any luck to see it maintained, but I cannot create issue on this one https://github.com/IntellectualSites/plot2dynmap

Boy0001's has gone "AWOL" as said here https://www.spigotmc.org/threads/plot2dynmap.30990/page-4#post-3262866

**Comment by AeSix** [comment]
no. please stop. Use the correct repo and issue tracker.

**Comment by MattBDev** [comment]
@TNTUP According to City, that repo isn't maintained and Jesse has control of the resource. AeSix just wasn't aware of that and I only just found out the other day.

**Comment by TNTUP** [comment]
> @TNTUP According to City, that repo isn't maintained and Jesse has control of the resource. AeSix just wasn't aware of that and I only just found out the other day.

Okay! Anyways, thanks guys ^^

**Comment by AeSix** [comment]
This is from City, I apologize if I mistook what he said, or misunderstood. 
```
11:14] Citymonstret: I pushed my changes to Jesse's repo of the addon so that's what we should use tbh
```

**Comment by TNTUP** [comment]
AeSix, its fine^^ I'm forgivin' you! Let's move on and enjoy PlotSquared 👍
